docs/Variables.adoc

```diff
@@ -30,13 +33,21 @@ Required variable, but may be set to `none` where not applicable.
 | Name
 | Description
 
+| `pass:[__meta__]`
+| Metadata used by the Babylon infrastructure when running AgnosticD.
+This variable should be ignored by workloads, configs, and roles.
+
 | `agonsticd_callback_token`
 | Authentication Bearer token used to callback with information and data collected from `agnosticd_user_info`.
 This is used to integrate AgnosticD with external systems including Babylon and GUID Grabber.
-Default value `__meta__.callback.token`.
+Default value `pass:[__meta__.callback.token]`.
 
 | `agonsticd_callback_url`
 | API URL used to callback with information and data collected from `agnosticd_user_info`.
 This is used to integrate AgnosticD with external systems including Babylon and GUID Grabber.
-Default value `__meta__.callback.url`.
+Default value `pass:[__meta__.callback.url]`.
+
+| `agnosticv_meta`
+| Metadata used by the OpenTLC and RHPDS admin host when running AgnosticD.
+This variable should be ignored by workloads, configs, and roles.
```

Comment:
is this a core variable ? if it should be ignore, where is it used ?
What does it do ?

Reply:
Same here. A developer may notice it is set and be tempted to use it, but all that I want to say here is don't use it, don't worry about it.